source/operations/data-recovery/recover-after-site-failure.rst

```diff
@@ -1,13 +1,76 @@
 .. _minio-restore-hardware-failure-site:
 
-==========================
-Recover after Site Failure
-==========================
+=====================
+Site Failure Recovery
+=====================
 
 .. default-domain:: minio
 
 .. contents:: Table of Contents
    :local:
    :depth: 1
 
-ToDo- site replication recovery procedure
+MinIO can make the loss of an entire site, while significant, a relatively minor incident.
+Site recovery depends on the backup option you use for the site.
+
+Site Replication
+----------------
+
+:ref:`Site replication <minio-site-replication-overview>` keeps two or more MinIO deployments in sync.
+If you suffer complete site loss with site replication enabled, you can restore IAM policies, objects, metadata, and other :ref:`replicable data <minio-site-replication-what-replicates>` from the remaining site(s).
```

Comment:
'site loss with site replication enabled' is a little clunky. Spitballing:

"If a peer site completely fails, such as due to a major disaster or long power outage, you can use the remaining healthy peer sites to restore..."

Reply:
Agreed.